api/nodes/views.py

```diff
@@ -264,6 +265,28 @@ def bulk_destroy_skip_uneditable(self, resource_object_list, user, object_type):
 
         return {'skipped': skipped, 'allowed': allowed}
 
+    # Overrides BulkDestroyModelMixin
+    def perform_bulk_destroy(self, resource_object_list):
+
+        auth = get_user_auth(self.request)
+        date = timezone.now()
+        id_list = [x.id for x in resource_object_list if x.can_edit(auth=auth)]
```

Comment:
Do we need the can_edit check if we are just using this id_list now for getting a queryset of AbstractNodes later on in this method?  I would assume I could edit every object in this list, because allow_bulk_destroy_resource requires the user to have admin permissions for every node.

Reply:
You are absolutely right. Nice catch. Thank you.